backend/nodes/image_dim_nodes.py

```diff
@@ -194,3 +194,36 @@ def run(
         result = img[top : h - bottom, left : w - right]
 
         return result
+
+
+@NodeFactory.register("chainner:image:get_dims")
+class GetDimensionsNode(NodeBase):
+    """Node for getting the dimensions of an image"""
+
+    def __init__(self):
+        """Constructor"""
+        super().__init__()
+        self.description = (
+            "Get the Height, Width, and number of Channels from an image."
+        )
+        self.inputs = [
+            ImageInput(),
+        ]
+        self.outputs = [
+            NumberOutput("Height"),
+            NumberOutput("Width"),
```

Comment:
I think we should use the same order as the Resize node.

```suggestion
            NumberOutput("Width"),
            NumberOutput("Height"),
```